cli/src/getDocusaurusConfig.ts

```diff
@@ -7,6 +7,7 @@ export interface GenerateConfigParams {
   enablePlugins: FoldersEnabled
   config: Config
   customCssExists: boolean
+  customSidebarsExists: boolean
```

Comment:
"custom sidebars exists" doesn't sound quite right... can we change it to `customSidebarExists` or `customSidebarsExist`?